openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -30,7 +32,7 @@ class CourseOverview(TimeStampedModel):
     """
 
     # IMPORTANT: Bump this whenever you modify this model and/or add a migration.
-    VERSION = 1
+    VERSION = 3
```

Comment:
The `VERSION` should be bumped up to 2, not 3.
If you are running into issues on your local devstack, then simply delete the "course_overviews" table in the Sql database to reset it.